src/protobot_hardware/launch/rs_d435_camera_with_model.launch

```diff
@@ -1,4 +1,5 @@
 <launch>
+  <remap from="/camera/color/image_raw/compressed" to="/camera/compressed"/>
```

Comment:
why, /camera/color/image_raw/compressed makes complete sense since it's the compressed version of image_raw, /camera/compressed is ambiguous and inflexible to the eventual separation of t265 and d435i namespaces

Reply:
can't speak much on the first fiducial file but I suspect this is due to unsaved changes after restoring from stash, getting rid of our prior remapping. will get this remapped right in the next commit.